scriptworker/task.py

```diff
@@ -293,7 +293,7 @@ def get_and_check_project(valid_vcs_rules, source_url):
         str: the project.
 
     """
-    project_path = match_url_regex(valid_vcs_rules, source_url, match_url_path_callback)
+    project_path = match_url_regex(valid_vcs_rules, source_url, match_url_path_callback, trusted=False)
```

Comment:
I don't see us doing anything with `trusted`.

Reply:
I could instead introduce a new config that is used here, instead of repurposing `valid_vcs_rules`. It was slightly more convenient to develop it this way (particularly before I realized this function wasn't needed for mobile

Reply:
Done.